cli/src/main/java/org/jboss/as/cli/handlers/ifelse/ExpressionParser.java

```diff
@@ -138,6 +138,8 @@ protected Operand parseOperand() throws CommandLineException {
                     } else {
                         --depthCount;
                     }
+                } else if (input.charAt(endIndex) == '"') { // traverse over string operands surrounded by quotes
```

Comment:
@aloubyansky any thoughts on this?

My intention was to allow escaping only _inside_ quote marks - everything properly enclosed in '"' marks would be skipped by this loop. If someone tries to escape '(' or ')' outside of "string" or a starting quote, then I believe behaviour will not change from current one.